dashboard/app/views/peer_reviews/index.html.haml

```diff
@@ -0,0 +1,33 @@
+%h1 Listing peer_reviews
```

Comment:
Assuming this is just an admin page, right?

If so - restrict access to admins only for the time being. Only admins can index, and maybe only do anything for the time being